command/whitelist/deployment/deployment.go

```diff
@@ -0,0 +1,66 @@
+package deployment
+
+import (
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/spf13/cobra"
+)
+
+func GetCommand() *cobra.Command {
+	deploymentCmd := &cobra.Command{
+		Use:     "deployment",
+		Short:   "Top level command for updating smart contract deployment whitelist, Only accepts subcommands",
+		PreRunE: runPreRun,
+		Run:     runCommand,
+	}
+
+	setFlags(deploymentCmd)
+
+	return deploymentCmd
+}
+
+func setFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(
+		&params.genesisPath,
+		chainFlag,
+		fmt.Sprintf("./%s", command.DefaultGenesisFileName),
+		"the genesis file to update",
+	)
+	cmd.Flags().StringArrayVar(
+		&params.addAddressRaw,
+		addAddressFlag,
+		[]string{},
+		"adds a new address in the contract deployment whitelist",
```

Comment:
Typo here, should be `adds a new address to`

Reply:
fixed 34e3e80bedc887edc0e753ed320b69617e050ad1